### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by GrahamCorker**

### CODE REVIEW

File: src/jsMain/kotlin/SignupComponent.kt [hunk 1]
```diff
@@ -1,20 +1,204 @@
...
 
 //Do we need this line below?
 //private val scope = MainScope()
 
 val SignupPage = functionalComponent<RProps> { _ ->
     val (username, setUsername) = useState("")
     val (password, setPassword) = useState("")
-    val (reEnterPassword, setReEnterPassword) = useState("")
+    val (email, setEmail) = useState("")
+    val (Zipcode, setZipcode) = useState("")
```
Comment (GrahamCorker) [inline src/jsMain/kotlin/SignupComponent.kt]: Can you make this zipcode and setZipcode in order to match the standard set elsewhere

File: src/jsMain/kotlin/SignupComponent.kt [hunk 2]
```diff
@@ -1,20 +1,204 @@
...
+        }
+        button{
+        +"Sign up"
+        attrs.onClickFunction = {
+            render(document.getElementById("root")) {
+                child(LoginPage)
+            }
+        }
+    }}
+    /*p{
```
Comment (GrahamCorker) [inline src/jsMain/kotlin/SignupComponent.kt:175]: Should this be here or be commented out?

File: src/jsMain/kotlin/SignupComponent.kt [hunk 3]
```diff
@@ -1,20 +1,204 @@
...
 import react.dom.*
 import kotlinext.js.*
+import kotlinx.browser.document
 import kotlinx.html.js.*
 import kotlinx.coroutines.*
 import kotlinx.css.*
+import smallComponents.InputComponent
 import styled.*
+import org.w3c.dom.HTMLSelectElement
+import org.w3c.dom.events.Event
```
Comment (GrahamCorker) [inline src/jsMain/kotlin/SignupComponent.kt]: remove unused imports

File: src/jsMain/kotlin/SignupComponent.kt (hunk 1 above)
Comment (tluo17) [reply src/jsMain/kotlin/SignupComponent.kt]: Got it

File: src/jsMain/kotlin/SignupComponent.kt (hunk 3 above)
Comment (tluo17) [reply src/jsMain/kotlin/SignupComponent.kt]: Already removed

File: src/jsMain/kotlin/SignupComponent.kt (hunk 2 above)
Comment (tluo17) [reply src/jsMain/kotlin/SignupComponent.kt:175]: It should be there.

**Review by rider-griffin — APPROVED** [review]

**Pull request merged**